.github/workflows/ci.yaml

```diff
@@ -74,19 +58,210 @@ jobs:
         cd build
         cmake .. \
           -DCMAKE_INSTALL_PREFIX:PATH=~/dependencies/kokkos_kernels \
+          -DCMAKE_BUILD_TYPE=Release
+        sudo cmake --build . --target install -j 2
+    - name: Clone
+      uses: actions/checkout@v3
+      with:
+        submodules: true
+    - name: Test OpenTurbine
+      run: |
+        export KokkosKernels_DIR=~/dependencies/kokkos_kernels
+        mkdir build
+        cd build
+        cmake .. \
+          -DOTURB_ENABLE_TESTS:BOOL=ON \
+          -DCMAKE_BUILD_TYPE=Release
+        cmake --build . -j 2
+        ./openturbine -h
+        ./openturbine_unit_tests
+  Debug:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install dependencies
+      run: sudo apt-get install libgtest-dev 
+    - name: Cache install Kokkos
```

Comment:
The most recent batch of commits have fixed the boiler-plate problem.  We should be able to avoid adding much in the future by using the matrix feature and disabling certain configurations to prevent combinatorial explosion.

Reply:
Ah that is great, thanks for updating